src/utils/validation.js

```diff
@@ -0,0 +1,32 @@
+export const validationFunction = {
+  userName: username => {
+    if (username === '') {
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
=== 자체가 true or false를 반환합니다!
```suggestion
      return !(username === '')
```
그리고 빈 문자열은 falsy값입니다!
```suggestion
    return !!username
```

Reply:
나머지도 === 이랑 if else 조합하신 부분 모두 마찬가지입니다!

Reply:
수정하였습니다!

Reply:
수정했습니다!